helm/prometheus-operator-app/values.yaml

```diff
@@ -28,16 +28,57 @@ prometheus-operator-app:
       ingressClassName: nginx
   kubeApiServer:
     enabled: true
+    serviceMonitor:
+      relabelings:
+      # Add app label.
+      - targetLabel: app
+        replacement: kubernetes
+      metricRelabelings:
+        # Keep from upstream values (https://github.com/prometheus-community/helm-charts/blob/kube-prometheus-stack-44.4.1/charts/kube-prometheus-stack/values.yaml#L967)
+      - action: drop
+        regex: apiserver_request_duration_seconds_bucket;(0.15|0.2|0.3|0.35|0.4|0.45|0.6|0.7|0.8|0.9|1.25|1.5|1.75|2|3|3.5|4|4.5|6|7|8|9|15|25|40|50)
+        sourceLabels:
+        - __name__
+        - le
+      # GS addition to reduce cardinality
+      - action: drop
+        regex: apiserver_request_slo_duration_seconds_bucket
+        sourceLabels:
+        - __name__
   kubelet:
     enabled: true
   kubeControllerManager:
     enabled: true
+    serviceMonitor:
+      https: true
+      insecureSkipVerify: true
+      relabelings:
+      # Add app label.
+      - targetLabel: app
+        replacement: kube-controller-manager
+      # Add node label.
+      - sourceLabels:
+        - __meta_kubernetes_pod_node_name
+        targetLabel: node
+
   kubeEtcd:
     enabled: true
   kubeProxy:
-    enabled: true
+    enabled: false
```

Comment:
I think we can keep this one on the other side for now so customers can enable it if they want right?

Reply:
Well in the app, true, in the bundle false? Because the situation about kube-proxy is a blur right now. WDYT?

Reply:
READY